lib/src/components/NxCodeSnippet/NxCodeSnippet.scss

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-variables';
+.nx-code-snippet {
+  align-items: end;
+  display: grid;
+  grid-template:
+    "label    copy-btn" 1fr
+    "sublabel copy-btn" auto
+    "textarea textarea" auto /
+      1fr      auto;
+
```

Comment:
Speaking for myself I really dislike this shorthand. I'd prefer to see this broken out into `grid-template-areas`, `grid-template-rows`, and `grid-template-columns` for clarity.

Reply:
> Speaking for myself I really dislike this shorthand. I'd prefer to see this broken out into grid-template-areas, grid-template-rows, and grid-template-columns for clarity.

Honestly I find it clearest this way, at least when using line breaks in the way that I have.  To me it makes it easy to visualize the template spec as a table, where for instance the `1fr` at the bottom can be easily understood as the width of the column above it.  When people use this syntax but don't align things though it can get very confusing.